dt-backend/test/model/user/user.e2e-spec.ts

```diff
@@ -16,32 +14,61 @@ describe('UserController e2e', () => {
     await app.init();
   });
 
-  it('/GET ', () => {
-    return request(app.getHttpServer()).get('/user').expect(200).expect('[]');
-  });
-
-  it('/user/signUp ', () => {
+  test('/update/:id', async () => {
+    // create user
     const user = {
       username: 'john',
       password: 'john',
       mail: 'john@john.com',
     };
-    return request(app.getHttpServer())
+    //send create request for john
+    const res = await request(app.getHttpServer())
       .post('/user/signUp')
       .send(user);
+
+    //update user john to joe
+    const updateUser = {
+      username: 'joe',
+      password: 'joe',
+      mail: 'joe@joe.com',
+    };
+    const update = await request(app.getHttpServer())
+      .patch('/user/update/' + res.body.id)
+      .send(updateUser);
+
+    //delete user
+    await request(app.getHttpServer())
+      .delete('/user/delete/' + res.body.id)
+      .send(res.body.id);
   });
```

Comment:
All you're doing here is going through the whole `update` workflow once. This is great, but you also need to test whether everything works as expected! You basically need to test the responses, whether they contain the information needed!
Such as:
- after the update request is the response really changed to `joe, joe, joe@joe.com` or is it still the old one?
etc...

That works using assertions, i.e.: [expect(updateUser.body.username).toBe('joe')](https://jestjs.io/docs/expect#tobevalue)